tests/sqlparser_common.rs

```diff
@@ -889,14 +892,11 @@ fn parse_like() {
         );
         let select = verified_only_select(sql);
         assert_eq!(
-            Expr::BinaryOp {
-                left: Box::new(Expr::Identifier(Ident::new("name"))),
-                op: if negated {
-                    BinaryOperator::NotLike
-                } else {
-                    BinaryOperator::Like
-                },
-                right: Box::new(Expr::Value(Value::SingleQuotedString("%a".to_string()))),
+            Expr::Like {
```

Comment:
Could you please also extend the `parse_like` and `parse_ilike` tests to include coverage of `escape_char` for both `LIKE` and `ILIKE`?